drivers/crypto/Kconfig.smartbond

```diff
@@ -0,0 +1,25 @@
+# Smartbond Cryptographic Accelerator configuration options
+
+# Copyright (c) 2020 Markus Fuchs <markus.fuchs@de.sauter-bc.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig CRYPTO_SMARTBOND
+	bool "Smartbond Cryptographic Accelerator driver"
+	default n
+	help
```

Comment:
Probably want a `depends on` here with something

Reply:
Added device tree dependency as done with the rest of the smartbond drivers.